lib/kitchen/verifier/inspec.rb

```diff
@@ -132,7 +150,6 @@ def runner_options_for_ssh(config_data)
         }
         opts['key_files'] = kitchen[:keys] unless kitchen[:keys].nil?
         opts['password'] = kitchen[:password] unless kitchen[:password].nil?
-
         opts
```

Comment:
If you want to save space here, why not also drop the `opts`? It's the return value of the assignment anyways. And we might as well drop that assignment and just return the hash. ;)